pkg/cmd/run.go

```diff
@@ -46,35 +43,13 @@ Envoy state and machine state into the ` + "`~/.getenvoy/debug`" + ` directory.`
 		Example: fmt.Sprintf(`# Run using a manifest reference.
 getenvoy run %[1]s -- --config-path ./bootstrap.yaml
 
-# Run using a filepath.
-getenvoy run /usr/local/bin/envoy -- --config-path ./bootstrap.yaml
-
 # List available Envoy flags.
 getenvoy run %[1]s -- --help
-
-# Run with Postgres specific configuration bootstrapped
-getenvoy run postgres:nightly --templateArg endpoints=127.0.0.1:5432,192.168.0.101:5432 --templateArg inport=5555
 `, defaultreference.Latest),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if err := InitializeRunOpts(o, args[0]); err != nil {
```

Comment:
guess this panics when users execute `getenvoy run` (without args).

Reply:
yeah I want to swap out the cli lib, but I'll fix this here and now meanwhile.